Sources/PrinceOfVersions/Objective-C Helpers/ResponseModel Wrapers/AppStoreData/AppStoreUpdateInfoObjectiveC.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AppStoreUpdateInfoObjectiveC.swift
+//  PrinceOfVersions
+//
+//  Created by Ivana Mršić on 02/06/2020.
+//
+
+import Foundation
+
+@objcMembers
+public class AppStoreUpdateInfoObject: NSObject {
```

Comment:
Should we hide this for swift?

Reply:
Even though this class is written to be used with Objective-C PoV methods, exposed Swift methods to Objc are still written in Swift so we cannot hide this.

Reply:
I think that I had a problem with hiding this from swift as well.. the main problem is that if you add `@available(swift, obsoleted: 1.0)` mark, this won't be visible for PoV internally as well :/

Reply:
Your thoughts on using something like this to discourage usage in swift?
```
@objc(AppStoreUpdateInfoObject)
public class __ObjcAppStoreUpdateInfoObject: NSObject 
```

Reply:
🤔 tbh, this is not such a bad idea... @ivanamrsic we could go with that approach :)